dhis-2/dhis-services/dhis-service-administration/src/main/java/org/hisp/dhis/resourcetable/jdbc/JdbcResourceTableStore.java

```diff
@@ -211,4 +211,22 @@ public void batchUpdate( int columns, String tableName, List<Object[]> batchArgs
 
         jdbcTemplate.batchUpdate( builder.toString(), batchArgs );
     }
+
+    @Override
+    public List<Integer> getAvailableDataYears()
```

Comment:
I am wondering if the JdbcResourceTableStore is not the right place for this method, as it does not relate to resource tables?